### PR TITLE
dm-2849 - Privacy Policy link in footer

### PR DESCRIPTION
**Pull request by bradjohnson92008**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2849

## Description - what does this code do?
This change removes "Nominate a practice" from the DM footer and adds "Privacy policy" link (https://www.va.gov/privacy-policy) to footer.  Link to va.gov privacy policy opens in a new tab.

## Testing done - how did you test it/steps on how can another person can test it 
1,  Login in to DM.
2,  Scroll down to the footer.  You will see the "Privacy policy" link in the footer.  
image

3.  Click on the link.  VA.gov privacy will display in new tab.
image

4.  Note: the "Nominate a practice" footer link is removed.

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
N/A

## Acceptance criteria
- Add "Privacy Policy" to footer
- Have it link to https://www.va.gov/privacy-policy/
- Check with Becca on which order it should go in

## Definition of done
- [X ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [X ] Documentation has been updated, if applicable
- [X ] A link has been provided to the originating JIRA issue
- [X ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -29,8 +29,8 @@
         <a id="feedback-modal" class="feedback-survey-popup desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-5 grid-col-12 padding-y-2 padding-x-2 text-bold dm-footer-link" href="#">
           Send feedback
         </a>
-        <a class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-5 grid-col-12 padding-y-2 padding-x-2 text-bold dm-footer-link" href="<%= nominate_a_practice_path %>" >
-          Nominate a practice
+        <a href="https://www.va.gov/privacy-policy" target="_blank" class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-5 grid-col-12 padding-y-2 padding-x-2 text-bold dm-footer-link" aria-controls="dm-terms-and-conditions-modal">
```
Comment (JoshingYou1) [inline app/views/shared/_footer.html.erb:32]: Can we remove the `aria-controls='dm-terms-and-conditions-modal'`?

**Review by JoshingYou1 — CHANGES_REQUESTED** [review]
Looks great to me Brad. Just one small thing I noticed. Nice job!

File: app/views/shared/_footer.html.erb (hunk 1 above)
Comment (bradjohnson92008) [reply app/views/shared/_footer.html.erb]: Good catch Josh - I removed it.

**Review by JoshingYou1 — APPROVED** [review]
Looks good to me Brad. Nice work!

**Pull request merged**